grin/src/Transformations/Syntax.hs

```diff
@@ -0,0 +1,164 @@
+{-# LANGUAGE MultiParamTypeClasses #-}
+{-# LANGUAGE OverloadedStrings #-}
+module Transformations.Syntax where
```

Comment:
Why is the conversion between new and old syntax regarded as a program transformation? I feel this code should live under a new module space like `Conversion`

Reply:
I agree.

Reply:
Done.